packages/@expo/cli/src/start/server/middleware/suppressErrorMiddleware.ts

```diff
@@ -1,17 +0,0 @@
-import type { IncomingMessage, ServerResponse } from 'http';
```

Comment:
Nice!

Reply:
We don't need to suppress what doesn't exists 😄